server/keyshare/tasks/tasks.go

```diff
@@ -66,6 +76,24 @@ func runWithTimeout(fn func(ctx context.Context)) error {
 	return ctx.Err()
 }
 
+func hasEmailRevalidation(conf *Configuration, db *keyshare.DB) bool {
+	ctx, cancel := context.WithTimeout(context.Background(), taskTimeout)
+	defer cancel()
+
+	c, err := db.ExecCountContext(ctx, "SELECT true FROM information_schema.columns where table_schema='irma' AND table_name='emails' AND column_name='revalidate_on'")
+	if err != nil {
+		conf.Logger.WithField("error", err).Error("Could not query the schema for column emails.revalidate_on, therefore revalidation is disabled")
+		return false
+	}
+
+	if c == 0 {
+		conf.Logger.Warning("Email address revalidation is disabled because the emails.revalidate_on column is not present in the schema")
+		return false
+	}
+
+	return true
+}
```

Comment:
It looks like `myirmaserver` has a similar function. You could consider moving this to being a method of the `keyshare.DB` struct, such that both `myirmaserver` and `tasks` can use this implementation.

Reply:
This code is still duplicated. Why not move this code to `./server/keyshare/db.go`? Then both myirmaserver and tasks can call this.

Reply:
You where a little too quick, had almost finished it already;) It is now pushed and the whole thingy is ready for review again